### DISCUSSION TITLE
Is it possible to have a VIEW field calculate and display the result of INPUT[number]?

### DISCUSSION THREAD

**Discussion by dmantisk** [mProjectsCode/obsidian-meta-bind-plugin#482] [Q&A] [answered]
Example
`INPUT[number:memory^calc]`
`VIEW[{memory^calc}]`

If I input a number in INPUT, it will be displayed in VIEW.
![image](https://github.com/user-attachments/assets/560a3669-3b99-4247-8bdc-7a0b87d68ec8)

But if I add +, - etc. VIEW will simply not work.
![image](https://github.com/user-attachments/assets/69818500-dd03-4681-bf53-77f17dda2016)

I wish to use it inside a modal window where I won't be able to edit the source for the VIEW field. That's why I'm trying to do it this way.

**Comment by mProjectsCode** [comment, marked as answer]
No. Bind targes in view fields are treated as variables to evaluate a formula. This is done for performance reasons. Most notably so that the formula must only be parsed once.

**Reply by dmantisk** [reply]
Thank you, especially for the explanation
